lib/dfe_sign_in/api/user.rb

```diff
@@ -58,20 +58,32 @@ def has_role?(role_code)
       end
 
       def role_codes
+        unless service_access?
+          raise ExternalServerError, "#{response.code}: #{response.body}" unless response.code.eql?("200")
```

Comment:
is the `unless` needed given the line above (61) is guarding already?

Reply:
No, I'll put a fix for that in a future PR